static/src/stylesheets/head.survey.scss

```diff
@@ -40,3 +40,9 @@
    ========================================================================== */
 
 @import 'module/atoms/_quiz';
+
+/* ==========================================================================
+   Survey
+   ========================================================================== */
+
+@import 'module/survey/survey';
```

Comment:
@joelochlann Does placing it here now mean we get all these other styles we don't need?